app/assets/javascripts/media_player_wrapper/avalon_player_new.es6

```diff
@@ -226,15 +242,19 @@ class MEJSPlayer {
     let defaults = {
       alwaysShowControls: true,
       pluginPath: "/assets/mediaelement/shims/",
-      features: ['playpause', 'current', 'progress', 'duration', 'volume', 'quality', 'fullscreen'],
+      features: ['playpause', 'current', 'progress', 'duration', 'volume', 'quality', 'fullscreen', 'addToPlaylist'],
```

Comment:
Please put addToPlaylist before fullscreen.  The order of this array indicates the order the controls appear and we always want fullscreen on the far right.

Reply:
Got it.  Updating now.